src/to-fresh.js

```diff
@@ -71,6 +71,7 @@ Convert JRS resume sections to FRESH.
             start: edu.startDate,
             end: edu.endDate,
             grade: edu.gpa,
+            title: edu.title,
```

Comment:
These two come close.  However, they don't resemble a title, right?  To me it sounds more like a hack to put them together.